tests/spectron/spellChecker.spectron.js

```diff
@@ -79,8 +79,8 @@ describe('Tests for spellChecker', () => {
     it('should invoke context menu ', (done) => {
         if (isMac) {
             app.browserWindow.getBounds().then((bounds) => {
-                let x = bounds.x + 45;
-                let y = bounds.y + 398;
+                let x = bounds.x + 55;
```

Comment:
please add a code comment as to why the bounds are being hard coded. Also, can you investigate if there is a smarter way of doing this than hard coding it?